app/src/androidTest/java/com/github/sdpsharelook/SRAlgo/SRAlgoTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.github.sdpsharelook.SRAlgo
+
+import android.content.Context
+import android.provider.Telephony.Mms.Part.FILENAME
+import androidx.test.core.app.ApplicationProvider
+import com.github.sdpsharelook.revision.SRAlgo
+import com.github.sdpsharelook.revision.revisionWord
+import org.junit.Before
+import org.junit.Test
+import java.io.File
+import java.io.FileOutputStream
+
+class SRAlgoTest {
```

Comment:
Please dont use SR for Spaced repetition since i already used this acronyme for speechrecognizer. I will also modify it in my previous code